misc/session_info.qmd

```diff
@@ -0,0 +1,24 @@
+---
+title: Copyright
+exclude-listing: true
+---
+
+## Session Info
+
+```{r sessioninfo}
+#| code-fold: show
+#| message: false
+sessionInfo()
+```
+
+---
+
+## Biomarker Analysis Catalog Version
+
+```{r}
+#| echo: false
+version <- read.dcf("../DESCRIPTION")[1, ]["Version"]
```

Comment:
For now is good, I wonder if in next iteration it could make sense to have a supporting R package or Python module that has those parsing helpers provided for BAC and TLGC and other catalogues to follow